Source/Core/Core/PatchEngine.cpp

```diff
@@ -211,6 +224,52 @@ static void ApplyPatches(const std::vector<Patch>& patches)
   }
 }
 
+std::unique_ptr<std::map<u64, std::vector<u8>>> CalculateDiscPatches(const DiscIO::Volume &disc) {
```

Comment:
Any reason to make this a unique_ptr instead of just relying on copy ellision / moves?